alpha/external-web/server.js

```diff
@@ -65,6 +66,8 @@ app.use(bodyParser.json())
 app.use(bodyParser.urlencoded({
   extended: true
 }))
+// Support for express-validator
+app.use(expressValidator())
```

Comment:
want to check again if we need to use express-validator over validator, as we do not need to use it as middleware